eng/Versions.props

```diff
@@ -156,7 +156,6 @@
     <MicrosoftAspNetCoreBlazorMonoPackageVersion>0.8.0-preview1-20181126.1</MicrosoftAspNetCoreBlazorMonoPackageVersion>
     <!-- When updating this, ensure you also update src/Components/Browser.JS/src/package.json to reference the corresponding version of @dotnet/jsinterop -->
     <MonoWebAssemblyInteropPackageVersion>0.8.0-preview1-20181126.4</MonoWebAssemblyInteropPackageVersion>
-    <MicrosoftJSInteropPackageVersion>0.8.0-preview1-20181126.4</MicrosoftJSInteropPackageVersion>
```

Comment:
The right version is defined here: https://github.com/aspnet/AspNetCore/compare/prkrishn/jsinterop?expand=1#diff-8b8f08ffbf7b863fb3700c1718eeb4cbR96